specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2019-04-01/managedClusters.json

```diff
@@ -1822,23 +1822,6 @@
       },
       "description": "Identity for the managed cluster."
     },
-    "OrchestratorProfile": {
```

Comment:
This is a breaking change for a stable version. Were any SDKs published for this spec?

Reply:
@dsgouda thanks for noting this. I don't believe spec 2019-04-01 has been published yet, so it should be ok.
@xizhamsft can you confirm that no SDKs were published for 2019-04-01? Also, Xiaofang, I removed it because I couldn't find any references to it

Reply:
@dsgouda , We have not published the SDK using 2019-04-01 api-version. We do have SDK published for earlier api-versions like 2019-02-01. Would that be a problem?

Reply:
If no SDKs have been published, we can move forward with these changes